src/ceph_osd.cc

```diff
@@ -375,6 +375,7 @@ int main(int argc, const char **argv)
 	    << " for osd." << whoami
 	    << " fsid " << g_conf().get_val<uuid_d>("fsid")
 	    << dendl;
+    forker.exit(0);
```

Comment:
This change just adds clarity, without changing the logics.
Makes sense, but I'd like a second opinion.

Reply:
if `mkfs` is "true", we are going to exit in line 381. why do we need it here?

Reply:
@yuvalif 
> if `mkfs` is "true", we are going to exit in line 381. why do we need it here?

Since we have entered the loop and on line `381` we anyway are going to exit isn't it better to exit in 

```
if (mkfs) {
    common_init_finish(g_ceph_context);
    if (g_conf().get_val<uuid_d>("fsid").is_zero()) {
      derr << "must specify cluster fsid" << dendl;
      forker.exit(-EINVAL);
    }
    int err = OSD::mkfs(g_ceph_context, std::move(store), g_conf().get_val<uuid_d>("fsid"),
                        whoami, osdspec_affinity);
    if (err < 0) {
      derr << TEXT_RED << " ** ERROR: error creating empty object store in "
	   << data_path << ": " << cpp_strerror(-err) << TEXT_NORMAL << dendl;
      forker.exit(1);
    }
    dout(0) << "created object store " << data_path
	    << " for osd." << whoami
	    << " fsid " << g_conf().get_val<uuid_d>("fsid")
	    << dendl;
}
```

and remove the `mkfs` in 
```
  if (mkfs || mkkey) {
    forker.exit(0);
  }
```